steps/test/unit/tReorderMSData.cc

```diff
@@ -21,6 +22,30 @@ using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
+bool compareMetaBinaries(const std::string& filename1,
+                         const std::string& filename2) {
+  std::ifstream file1(
+      filename1,
+      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+  std::ifstream file2(
+      filename2,
+      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+
+  if (file1.fail() || file2.fail()) {
+    std::cout << "Failed to open Meta file" << std::endl;
+  }
+
+  file1.ignore(std::numeric_limits<std::streamsize>::max(),
+               '\n');  // Ignore first line
```

Comment:
These comments don't help clarity at all.

Quote from google cpp style guidelines:
"Do not state the obvious. In particular, don't literally describe what code does, unless the behavior is nonobvious to a reader who understands C++ well. Instead, provide higher level comments that describe why the code does what it does, or make the code self describing."

So in this case, why is the first line of these files being ignored? This is whats interesting to me as someone reading the code, that it is being ignored is realtively easy to figure out from reading the code...

Rather remove the duplicate comments here and then place one single comment (before the first ignore) stating the purpose/reason for the ignore.